plugins/@grouparoo/snowflake/README.md

```diff
@@ -8,6 +8,19 @@ Learn more about Grouparoo Plugins at [www.grouparoo.com/docs/plugins](https://w
 
 In your Grouparoo project, run `grouparoo install @grouparoo/snowflake`.
 
+## Notes on formatting your .p8 private key when using a `snowflake-keypair` app:
+
+You will need to convert your `*.p8` into a single string so that Grouparoo can store it and parse it. You can do this with the `awk` command:
+
+```bash
+# If rsa_key.p8 is the private key you generated for snowflake auth, per https://docs.snowflake.com/en/user-guide/key-pair-auth.html#configuring-key-pair-authentication
+# The following command will output a string Grouparoo can use for the `Private Key` connection option
+
+awk 'NF {sub(/\n/, ""); printf "%s\\n",$0;}' rsa_key.p8
+
+# note: some operating systems will include a final `%` at the end of the line.  Do not not include it
+```
+
```

Comment:
This was a whole thing :/

Reply:
eh... I'm not too worried about it.  The /value/ won't be stored in your shell history.  And if we wrote to a file, the next step would be to `cat` it anyway so 🤷